integration-tests/contract-confirmation-timeout.spec.ts

```diff
@@ -0,0 +1,40 @@
+import BigNumber from "bignumber.js";
+import { CONFIGS } from "./config";
+
+const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms));
```

Comment:
since we used this more than a couple times now, perhaps we should create a utils.ts file in `integration-tests`?

Reply:
Make sense, I will create the utils file, thanks.

Reply:
Moved to the config.ts file instead